### DISCUSSION TITLE
Disabling telemetry in setting

### DISCUSSION THREAD

**Discussion by AuthorShin** [coder/code-server#5117] [Q&A] [answered]
Hello
Is turning telemetry off in Settings > User > Application > Telemetry is same as using the '--disable-telemetry' flag? (Even the extension ones? #1116)
Cause I'm using [CasaOS](https://github.com/IceWhaleTech/CasaOS) to install Code-server and as far as I know you can not add commands nor flags with it.

**Comment by jsjoeio** [comment, marked as answer]
I'm not 100% sure...I think they're slightly different. I think the one in the Settings tells extensions you don't want telemetry on. 

I'm not familiar with CasaOS. Can you use a custom `config.yaml` with your code-server instance? You should be able to put "disable-telemetry: true" in that.

**Reply by AuthorShin** [reply]
I can add Environment variables if disable-telemetry is supported for that. like: disable-telemetry=true
Does it?

**Comment by AuthorShin** [comment]
Is this gonna do the just? (I edit this after the installation is it okay?) what about setting it as a environment variable?
<a href="https://imgbb.com/"><img src="https://i.ibb.co/MNMPgvK/Screenshot-2022-04-24-005338.png" alt="Screenshot-2022-04-24-005338" border="0"></a>

**Reply by jsjoeio** [reply]
This should do it! Interesting...the helm chart has an environment variable for it [here](https://github.com/coder/code-server/blob/dea6a40ea8e5601949d8884760a83a04b5294e26/ci/helm-chart/values.yaml#L71-L72) but I am not 100% sure it will work in the standalone or other installations of code-server

**Reply by AuthorShin** [reply]
@jsjoeio Thank you very much for the answer, I don't know of a way to test it after adding as environment variable so cannot prove if it's working that way, so maybe you got a way for it?
By the way I use lscr.io/linuxserver/code-server:latest image so don't know if it's any different or not.

**Reply by jsjoeio** [reply]
Ah gotcha! Well in that case, I'd use the `config.yaml` and call it a day 🎉
